docs/services/base-rc/_index.md

```diff
@@ -25,6 +25,15 @@ Control mode is determined by the configuration attribute `"mode"`, for which th
 
 You can monitor the input from these controls in the **Control** tab of the [Viam app](https://app.viam.com).
 
+## Required Components
```

Comment:
something is strange here, when you configure it there is no movement sensor option and also no board. I think the requirement are base and input controller instead